### PR TITLE
Feature/mq refactoring

### PR DESCRIPTION
**Pull request by farexy**

### CODE REVIEW

File: HdProduction.BuildService/Startup.cs [hunk 1]
```diff
@@ -19,20 +22,41 @@ public Startup(IConfiguration configuration)
     {
       Configuration = configuration;
     }
-    
+
```
Comment (Bogomazz) [inline HdProduction.BuildService/Startup.cs:25]: Looks funny)

File: HdProduction.BuildService/Startup.cs [hunk 2]
```diff
@@ -19,20 +22,41 @@ public Startup(IConfiguration configuration)
...
       services.AddScoped<IBuildsRepository, BuildsRepository>();
+
+      AddMessageQueue(services, Configuration.GetSection("MessageQueue"));
+    }
+    
+    private static void AddMessageQueue(IServiceCollection services, IConfigurationSection mqConfigurationSection)
+    {
+      if (mqConfigurationSection.GetValue<bool>("Enabled"))
+      {
+        services.AddSingleton<IRabbitMqConnection>(
```
Comment (Bogomazz) [inline HdProduction.BuildService/Startup.cs:47]: Are there places in build-service where we will use RebitMqConnection directly?

Comment (farexy) [reply HdProduction.BuildService/Startup.cs:47]: no:)

Comment (Bogomazz) [reply HdProduction.BuildService/Startup.cs:47]: So, what this registration performs for?

Comment (farexy) [reply HdProduction.BuildService/Startup.cs:47]: we need this registration, because Consumer and Publisher use this dependency

**Pull request merged**